curvefs/src/client/common/config.cpp

```diff
@@ -196,15 +196,15 @@ void InitVolumeOption(Configuration *conf, VolumeOption *volumeOpt) {
                               &volumeOpt->allocatorOption.type);
 
     conf->GetValueFatalIfFail(
-        "volume.blockgroup.allocate_once",
+        "volume.blockGroup.allocate_once",
         &volumeOpt->allocatorOption.blockGroupOption.allocateOnce);
 
     if (volumeOpt->allocatorOption.type == "bitmap") {
         conf->GetValueFatalIfFail(
-            "volume.bitmapallocator.size_per_bit",
+            "volume.bitmapAllocator.size_per_bit",
```

Comment:
Should size_per_bit also use camel case?

Reply:
yes, for better reading, its good to unify the style. I will do this today.

Reply:
yes, for better reading, its good to unify the style. I will do this today.